mautic/mautic
# Public preview email disabled by default

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:

We noticed preview url of email is enabled by default.  But preview url is incrementally and anybody can pick up url of any email.  This PR add preview url disabled by default  


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create new email and check If preview url is disabled by default 
![image](https://user-images.githubusercontent.com/462477/153206782-12f104fa-55bf-4737-9849-b3df06b5b51e.png)




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10845"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
